packages/booter-lb3app/src/__tests__/acceptance/booter-lb3app.acceptance.ts

```diff
@@ -215,4 +215,17 @@ describe('booter-lb3app', () => {
       expect(createOp.summary).to.eql('just a very simple modification');
     });
   });
+
+  context('binding LoopBack 3 datasources', () => {
+    before(async () => {
+      ({app, client} = await setupApplication({
+        lb3app: {path: '../fixtures/app-with-model'},
+      }));
+    });
+
+    it('binds datasource to the context', async () => {
+      const dsBindings = app.findByTag('datasource');
+      expect(dsBindings[0].key).to.equal('datasources.lb3-memory');
```

Comment:
Can you verify that the binding value is equal to lb3app's `memory` datasource?

I think you can obtain that datasource instance by loading the app via `require()`.

```ts
const expected = require('../fixtures/app-with-model').dataSources.memory;
// ...
```